site/docs/hive.md

```diff
@@ -17,113 +17,211 @@
+
+#### Hive catalog tables
+
+As described before, tables created by the `HiveCatalog` with Hive engine feature enabled are directly visible by the Hive engine, so there is no need to create an overlay.
+
+#### Custom catalog tables
+
+For a registered catalog, simply specify the catalog name in the statement using table property `iceberg.catalog`.
+For example, the SQL below creates an overlay for a table in the `glue` catalog.
+
+```sql
+CREATE EXTERNAL TABLE database_a.table_c
+STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'
+TBLPROPERTIES ('iceberg.catalog'='glue');
+```
+
+### DROP TABLE
+
+When dropping the Hive table overlay, one can perform the following configurations:
+
+| Config key                  | Default                    | Description                                            |
+| ----------------------------| ---------------------------| ------------------------------------------------------ |
+| external.table.purge        | true                       | if all data and metadata should be purged in the table  |
+
+## Querying with SQL
+
+Here are the features highlights for Iceberg Hive read support:
+
+1. **Predicate pushdown**: Pushdown of the Hive SQL `WHERE` clause has been implemented so that these filters are used at the Iceberg `TableScan` level as well as by the Parquet and ORC Readers.
+2. **Column projection**: Columns from the Hive SQL `SELECT` clause are projected down to the Iceberg readers to reduce the number of columns read.
+3. **Hive query engines**: Both the MapReduce and Tez query execution engines are supported.
+
+You should now be able to issue Hive SQL `SELECT` queries and see the results returned from the underlying Iceberg table:
 
-#### Query the Iceberg table via Hive
-You should now be able to issue Hive SQL `SELECT` queries using the above table and see the results returned from the underlying Iceberg table.
 ```sql
-SELECT * from database_a.table_c;
+SELECT * from table_a;
 ```
 
-### Features
+## Writing with SQL
+
+### Configurations
 
-#### Predicate pushdown
-Pushdown of the Hive SQL `WHERE` clause has been implemented so that these filters are used at the Iceberg TableScan level as well as by the Parquet and ORC Readers.
+Here is a table of Hadoop configurations that one can adjust for the Hive writer:
 
-#### Column Projection
-Columns from the Hive SQL `SELECT` clause are projected down to the Iceberg readers to reduce the number of columns read.
+| Config key                                        | Default                                  | Description                                            |
+| ------------------------------------------------- | ---------------------------------------- | ------------------------------------------------------ |
+| iceberg.mr.output.tables                          | the current table in the job             | a list of tables delimited by `..`                     |
```

Comment:
`iceberg.mr.output.tables` should not be configurable by the user. Its value is assembled by the storage handler during runtime for each query